app/scripts/metamask-controller.js

```diff
@@ -1136,12 +1123,11 @@ module.exports = class MetamaskController extends EventEmitter {
    */
   setCurrentCurrency (currencyCode, cb) {
     try {
-      this.currencyController.setCurrentCurrency(currencyCode)
-      this.currencyController.updateConversionRate()
+      this.currencyRateController.currency = currencyCode
```

Comment:
Good point. I'm working on much more thorough documentation since the auto-generated API documentation is a bit dense. I'll be sure to hit on new concepts like setters, initialization flow, default state, default config, etc.